spec/latest/core/core.jsonld

```diff
@@ -442,19 +443,28 @@
       "@type": "rdf:Property",
       "label": "variable representation",
       "comment": "The representation format to use when expanding the IRI template.",
+      "range": "hydra:VariableRepresentation",
       "vs:term_status": "testing"
     },
     {
-      "@id": "hydra:BasicRepresentation",
+      "@id": "hydra:VariableRepresentation",
       "@type": "hydra:Class",
       "subClassOf": "hydra:Resource",
+      "label": "representation of variable values",
+      "comment": "A representation specifies how to serialize variable values into strings.",
+      "vs:term_status": "testing"
+    },
+    {
+      "@id": "hydra:BasicRepresentation",
+      "@type": "hydra:VariableRepresentation",
+      "subClassOf": "hydra:Resource",

```

Comment:
Those lines were still remains of the previous version; 9e267985fdc0ef7e84c5d19a0aae4bdeb5f565e6 removes them. It's probably not necessary to explicitly list `hydra:Resource` as type, since we now have `hydra:VariableRepresentation` as the more specific type.